src/functions/generateCallbackSort.ts

```diff
@@ -0,0 +1,56 @@
+export const generateCallbackSort = (sortBy: string, sortOrder: string) => {
+  const multiplier = sortOrder === 'asc' ? 1 : -1;
+  let out: number;
+
+  const fieldName = sortBy.toLowerCase();
+
+  // i don't know how set sortBy str as keyof
+  // HumanWithParents because i take that value from table headings
```

Comment:
U can modify headings or create enum with heading names

Reply:
and after event catch value and thhrow it in action like 'some_string as EnumHeadings'